src/repository/groupRepository.js

```diff
@@ -0,0 +1,27 @@
+import { Model } from "../db/models/index.js";
+import { ErrorHander } from "../utils/errorHander.js";
+
+class GroupRepository {
+  createGroup = async ({ group_name, desc }) => {
```

Comment:
this is not the rightway, insted of using this parametrs in function direcly used one parameter which is object
 createGroup = async (data, options) => {
const newGroup = await Model.GroupDetail.create(data, options);
}
options is condition or other inputs which we can send with request to sequelize which can include attributes etc.
do this for all repository functions